model-optimizer/extensions/middle/ReverseV2ToReverseSequence.py

```diff
@@ -57,14 +57,13 @@ def replace_pattern(self, graph: Graph, match: dict):
 
         # 1. For ReverseSequence 1-port input is seq_lengths => create this input node
         seq_lengths = np.ones(input_data_shape[batch_axis]) * input_data_shape[seq_axis]
-        const = Const(graph, dict(value=seq_lengths)).create_node()
 
         # 2. Create new ReverseSequence node and reconnect all inputs/outputs to it
-        reverse_sequence = ReverseSequence(graph, {'name': reverse.name + '/ReverseSequence/',
-                                                   'seq_axis': seq_axis, 'batch_axis': batch_axis}).create_node()
-
+        reverse_sequence = create_op_node_with_second_input(graph, ReverseSequence, seq_lengths,
```

Comment:
Call `rename_node` on a `reverse_sequence` to restore original node name.

Reply:
Done.